app/models/template_class_csv.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class TemplateClassCsv
```

Comment:
I don't see `TemplateClassCsv` used anyone else besides `Reports::TraineesReport`. Is it worth keeping?